src/hooks/use-fetch.js

```diff
@@ -25,10 +24,10 @@ function useFetch(url, postprocessData, debounceTime) {
             setError(errorMessage);
             setData(null);
         } else {
-            const parsedData = await postprocessData(resp);
-            if (parsedData) {
+            const jsonData = await resp.json();
```

Comment:
I wonder if se can skip this step altogether. Do you know if the api's we'll be using already return json?

Reply:
Ah [here](https://webhooks.mongodb-realm.com/api/client/v2.0/app/devhubsearch-hgphu/service/devhub_search/incoming_webhook/devhub_search?q=java) is the endpoint we will be using (at least next for the text filter project). It does give us json but maybe a more appropriate idea is to just return the response in this hook and also use `.json()` in another custom hook specifically for the text filter data manipulation. What do you think about that?

Reply:
Ah I see, yea that makes sense. If you find more cases of us needing to convert data to json down the line we can always put `.json()` back in this hook at that point.